pai-management/src/gpu-exporter/dockerfile

```diff
@@ -29,15 +29,17 @@ ENV PATH=$PATH:$NV_DRIVER/bin
 WORKDIR /root/
 
 RUN apt-get update && \
-    apt-get -y install wget build-essential python python-pip git pciutils
+    apt-get -y install wget build-essential python python-pip git pciutils iftop lsof
 
 COPY copied_file/exporter/* /usr/local/
 
 RUN wget https://download.docker.com/linux/static/stable/x86_64/docker-17.06.2-ce.tgz
 RUN cp docker-17.06.2-ce.tgz /usr/local
 RUN tar xzvf /usr/local/docker-17.06.2-ce.tgz -C /usr/local/
 RUN cp -r /usr/local/docker/* /usr/bin/
-
+RUN git clone https://github.com/yadutaf/infilter
```

Comment:
>t clone https://github.com/yadutaf/infilter [](start = 6, length = 43)

can we use nsenter instead?

Reply:
after i and yibo test. 

nsenter also need each job container install lsof (like docker exec cmd) if container does not contains lsof cmd. [The installation can called by exporter container]

Then yibo recommend infilter.